securedrop/config.py.example

```diff
@@ -17,6 +17,10 @@ class SourceInterfaceFlaskConfig(FlaskConfig):
 class JournalistInterfaceFlaskConfig(FlaskConfig):
     SECRET_KEY = '{{ journalist_secret_key.stdout }}'
     SESSION_COOKIE_NAME = "js"
+    SESSION_SIGNER_SALT = "js_session"
```

Comment:
New config.py variables should not be expected to be present on existing instances, as config.py is created at install time and rarely touched afterward.